test/test_helpers.rb

```diff
@@ -3,7 +3,7 @@
 require "rspecq"
 
 module TestHelpers
-  REDIS_HOST = "127.0.0.1".freeze
+  REDIS_HOST = {host: "127.0.0.1".freeze}.tap{|e| e.merge(url: ENV['REDIS_URL']) if ENV['REDIS_URL']}
```

Comment:
Let's rename this to `REDIS_OPTS` in order to be more accurate. And then we can simply do:

```ruby
REDIS_OPTS = { host: "127.0.0.1" }.freeze
```

I see no need to use `:url` here since the test environment is fixed and always runs in a local Redis.